Podfile

```diff
@@ -66,6 +75,11 @@ target 'Database' do
   pod 'Firebase/Database'
 end
 
+target 'Storage' do
```

Comment:
Well then... Any chance we could take some time (not now, but in the future, ideally before 1.0) to rename them to explain the difference (maybe `FirebaseUIStorageStatic` and `FirebaseUIStorageDynamic`)?